Source/Csla.Web.Mvc.Shared/Controller.cs

```diff
@@ -72,7 +72,7 @@ protected virtual async Task<bool> SaveObjectAsync<T>(T item, Action<T> updateMo
         updateModel?.Invoke(item);
         if (item is BusinessBase bb && !bb.IsValid)
         {
-          AddBrokenRuleInfo(item, null);
+          AddBrokenRuleInfo(item, string.Empty);
```

Comment:
Same concern as above as to how html renders null vs ""

Reply:
Same answer as above with regard to `null` handling in the underlying framework methods.